2018/5xxx/CVE-2018-5559.json

```diff
@@ -1,18 +1,130 @@
-      "ID" : "CVE-2018-5559",
-      "STATE" : "RESERVED"
-   },
-   "data_format" : "MITRE",
-   "data_type" : "CVE",
-   "data_version" : "4.0",
-   "description" : {
-      "description_data" : [
-         {
-            "lang" : "eng",
-            "value" : "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem.  When the candidate has been publicized, the details for this candidate will be provided."
-         }
+  "data_type": "CVE",
+  "data_format": "MITRE",
+  "data_version": "4.0",
+  "CVE_data_meta": {
+    "ID": "CVE-2018-5559",
+    "ASSIGNER": "cve@rapid7.com",
+    "DATE_PUBLIC": "2018-11-07T18:00:00.000Z",
+    "TITLE": "",
+    "AKA": "",
+    "STATE": "PUBLIC"
+  },
+  "source": {
+    "defect": [
+      "R7-2018-53"
+    ],
+    "advisory": "",
+    "discovery": "EXTERNAL"
+  },
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Rapid7",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Komand",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_name": "0.41.0",
+                      "affected": "<=",
+                      "version_value": "0.41.0",
+                      "platform": ""
+                    },
+                    {
+                      "version_name": "0.42.0",
+                      "affected": "!>=",
+                      "version_value": "0.42.0",
+                      "platform": ""
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
       ]
-   }
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "Certain endpoints that are able to list the always encrypted-at-rest connection data could return some configurations of connection data without obscuring sensitive data from the API response sent over an encrypted channel."
+      },
+      {
+        "lang": "eng",
+        "value": "This issue affects:\nRapid7 Komand version 0.41.0 and prior versions.\n\nThis issue does not affect:\nRapid7 Komand version 0.42.0 and later versions."
+      }
+    ]
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "CWE-212"
+          },
+          {
+            "lang": "eng",
+            "value": "Improper Cross-boundary Removal of Sensitive Data"
+          }
+        ]
+      }
+    ]
+  },
+  "references": {
+    "reference_data": [
+      {
+        "refsource": "CONFIRM",
+        "url": "https://docs.komand.com/docs/release-notes#section-komand-v0-42-0-2018-11-1",
+        "name": "https://docs.komand.com/docs/release-notes#section-komand-v0-42-0-2018-11-1"
+      },
+      {
+        "refsource": "CONFIRM",
```

Comment:
refsource should be MISC for the non-Rapid7 page: https://cve.mitre.org/data/refs/index.html#MISC

Reply:
Yep!

```suggestion
        "refsource": "MISC",
```